packages/expansion-hub/src/internal/ExpansionHub.ts

```diff
@@ -317,4 +322,19 @@ export class ExpansionHubInternal implements ExpansionHub {
     writeI2CSingleByte(i2cChannel: number, slaveAddress: number, byte: number): Promise<void> {
         return this.nativeRevHub.writeI2CSingleByte(i2cChannel, slaveAddress, byte);
     }
+
+    /**
+     * Listen for errors that can't be handled in the library
+     *
+     * @param eventName
+     * @param listener
+     */
```

Comment:
The `RevHub` version of this also needs a doc comment.

```suggestion
    /**
     * Listen for errors that do not happen as a result of a specific function call
     *
     * @param eventName
     * @param listener
     */
```

Reply:
Fixed